lib/topic.ts

```diff
@@ -0,0 +1,39 @@
+import { expect, Page } from '@playwright/test';
+import { waitForKafkaReady } from './kafka';
+
+export const navigateToKafkaTopicsList = async function (page: Page, kafkaName: string) {
+  await expect(page.getByText(kafkaName)).toHaveCount(1);
+  await waitForKafkaReady(page, kafkaName);
+  await page.getByText(kafkaName).click();
+  await page.getByText('Topics').click();
+};
+
+export const createKafkaTopic = async function (page: Page, name: string, check = true) {
+  await page.getByText('Create topic').click();
+  await expect(page.getByText('Create topic')).toHaveCount(2);
+  // This is default Topic creation
+  await page.getByPlaceholder('Enter topic name').fill(name);
+  await page.getByText('Next').click();
+  await page.getByText('Next').click();
+  await page.getByText('Next').click();
+  await page.getByText('Finish').click();
+
+  if (check) {
```

Comment:
this condition is never used, would you mind removing it?

Reply:
ok